storey/flow.py

```diff
@@ -629,3 +664,150 @@ def build_flow(steps):
             cur_step.to(next_step)
             cur_step = next_step
     return steps[0]
+
+
+class V3ioTable(NeedsV3ioAccess):
```

Comment:
Is this a utility class not used yet? If so, why expose it as part of the API?

Reply:
Discussed offline.
First, It is used. Second, decided not to expose it and infer it by a string of the storage name the user will pass.